TransIT.BLL/Services/IUserService.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using TransIT.DAL.Models.Entities;
+
+namespace TransIT.BLL.Services
+{
+    /// <summary>
+    /// Set a behavior of entities which act with users
+    /// </summary>
+    public interface IUserService
+    {
+        /// <summary>
+        /// Gets user by id
+        /// </summary>
+        /// <param name="userId">User id</param>
+        /// <returns>User</returns>
+        Task<User> Get(int userId);
+        /// <summary>
+        /// Gets users with pagination
+        /// </summary>
+        /// <param name="index">Amount to skip</param>
+        /// <param name="amount">Amount to take</param>
+        /// <returns>Users</returns>
+        Task<IEnumerable<User>> Get(uint index, uint amount);
+        /// <summary>
+        /// Registers a new user
+        /// </summary>
+        /// <param name="value">New user model</param>
+        /// <returns>Whether operation is successful</returns>
+        Task<bool> Create(User value);
```

Comment:
value => user?

Reply:
This method should return newly created user
Our action will contain user representation in the body